src/Sylius/Bundle/MoneyBundle/Templating/Helper/ConvertMoneyHelper.php

```diff
@@ -14,10 +14,11 @@
 namespace Sylius\Bundle\MoneyBundle\Templating\Helper;
 
 use Sylius\Component\Currency\Converter\CurrencyConverterInterface;
-use Symfony\Component\Templating\Helper\Helper;
 
-class ConvertMoneyHelper extends Helper implements ConvertMoneyHelperInterface
+class ConvertMoneyHelper implements ConvertMoneyHelperInterface
```

Comment:
`ConvertMoneyHelperInterface` missing new methods from HelperInterface.

Reply:
Hum I'm not totally sure, cause these two ones do not extend the FilterInterface, so it could be a bc-break.